### PR TITLE
[#889]

### PR DESCRIPTION
**Pull request by karbyshevds**
* Add Google Cloud Service Account Assigner Helm chart
* Add Legion data GCS bucket initialization and SA with access permissions
* Extend GKE nodes SA scope to allow GC Service Account Assigner work as expected

### CODE REVIEW

File: helms/k8s-gke-service-account-assigner/values.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+defaultScopes: "https://www.googleapis.com/auth/logging.write,https://www.googleapis.com/auth/monitoring,https://www.googleapis.com/auth/servicecontrol,https://www.googleapis.com/auth/service.management.readonly,https://www.googleapis.com/auth/trace.append"
+# defaultSA: ""
+# serviceAccountName: ""
+
+image:
+  repository: imduffy15/k8s-gke-service-account-assigner
+  tag: latest
```
Comment (alexander-semenets) [inline helms/k8s-gke-service-account-assigner/values.yaml]: we shouldn't bind to latest versions, please specify exact release version

File: helms/k8s-gke-service-account-assigner/templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "k8s-gke-service-account-assigner.name" -}}
```
Comment (alexander-semenets) [inline helms/k8s-gke-service-account-assigner/templates/_helpers.tpl]: can we transform "k8s-gke-service-account-assigner" to shorter more convenient name?

**Review by alexander-semenets — CHANGES_REQUESTED** [review]

File: helms/k8s-gke-service-account-assigner/values.yaml (hunk 1 above)
Comment (karbyshevds) [reply helms/k8s-gke-service-account-assigner/values.yaml]: fixed in 088bdb1

File: helms/k8s-gke-service-account-assigner/templates/_helpers.tpl (hunk 2 above)
Comment (karbyshevds) [reply helms/k8s-gke-service-account-assigner/templates/_helpers.tpl]: done in c03be05

**Review by alexander-semenets — APPROVED** [review]

**Pull request merged**